tools/deployment_manager/configs/linux.json

```diff
@@ -16,38 +16,38 @@
     "openvino_license": {
```

Comment:
the tools does not work:
```
ilya@odt-peter-backup:~/Documents/Programming/build/openvino-release/install/tools/deployment_manager$ python3 deployment_manager.py --targets cpu --output_dir ../../
[ 2023-09-27 02:42:09,993 ] WARNING : [ie_core] Missing /mnt/data3_1878/ilya/Documents/Programming/build/openvino-release/install/runtime/lib/intel64/libopenvino.so.2023.1.0
[ 2023-09-27 02:42:09,993 ] WARNING : [gna] Missing /mnt/data3_1878/ilya/Documents/Programming/build/openvino-release/install/runtime/lib/intel64/libgna.so.3.5.0.1906
[ 2023-09-27 02:42:09,995 ] WARNING : [ie_core] Missing /mnt/data3_1878/ilya/Documents/Programming/build/openvino-release/install/runtime/lib/intel64/libopenvino.so.2023.1.0
[ 2023-09-27 02:42:10,047 ] WARNING : [ie_core] Missing /mnt/data3_1878/ilya/Documents/Programming/build/openvino-release/install/runtime/lib/intel64/libopenvino.so.2023.1.0
Traceback (most recent call last):
  File "deployment_manager.py", line 25, in <module>
    sys.exit(main())
  File "/mnt/data3_1878/ilya/Documents/Programming/build/openvino-release/install/tools/deployment_manager/deployman/main.py", line 262, in main
    _manager.process()
  File "/mnt/data3_1878/ilya/Documents/Programming/build/openvino-release/install/tools/deployment_manager/deployman/main.py", line 165, in process
    dependency.copy_files(tmpdirname)
  File "/mnt/data3_1878/ilya/Documents/Programming/build/openvino-release/install/tools/deployment_manager/deployman/config.py", line 56, in copy_files
    raise FileNotFoundError("Files for component {} not found. "
FileNotFoundError: Files for component ie_core not found. Please check your OpenVINO installation
```
Please, add tests [here](https://github.com/openvinotoolkit/openvino/blob/master/.github/workflows/linux.yml#L233-L242) once you fix the tool. E.g. you can run the tool and the testing step should not fail with error at least